hbase-server/src/test/java/org/apache/hadoop/hbase/ipc/TestProtoBufRpc.java

```diff
@@ -81,10 +82,10 @@ public void setUp() throws IOException { // Setup server for both protocols
     this.conf = HBaseConfiguration.create();
     this.conf.set(RpcServerFactory.CUSTOM_RPC_SERVER_IMPL_CONF_KEY,
         rpcServerImpl);
-    Logger log = Logger.getLogger("org.apache.hadoop.ipc.HBaseServer");
-    log.setLevel(Level.DEBUG);
-    log = Logger.getLogger("org.apache.hadoop.ipc.HBaseServer.trace");
-    log.setLevel(Level.TRACE);
+    org.apache.log4j.Logger.getLogger("org.apache.hadoop.ipc.HBaseServer")
```

Comment:
Why is the full-qualified class name necessary?

Reply:
The slf4j logger uses exactly the same name 'Logger', using a full name is better to let developers know that this is not the slf4j one.

And I plan to forbid log4j imports in enforcer in later patches, then using the full name is the only way to reference log4j stuff.